eng/common/native/init-os-and-arch.sh

```diff
@@ -35,6 +35,10 @@ fi
 case "$CPUName" in
     arm64|aarch64)
         arch=arm64
+        if [ "$(getconf LONG_BIT)" -lt 64 ]; then
```

Comment:
make it more robust against failing getconf etc, since we'd rather use arm64 instead of arm in those edge cases:

```suggestion
        if [ "$(getconf LONG_BIT)" -eq 32 ]; then
```

Reply:
```suggestion
        if [ "$(getconf LONG_BIT 2>/dev/null || echo "64")" -eq 32 ]; then
```
if we really want to cover non-conforming edge-cases.

Reply:
I just realized that we already have this code in dotnet-install.sh so we'd have probably seen any issues by now. Feel free to ignore this.

Reply:
That's where I copied it from ... but the in-repo `dotnet-install.sh` is not used in the dotnet/runtime build.

Reply:
yeah, still better to be consistent with existing code :)